src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -15,9 +14,9 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @ManyToOne(fetch = FetchType.LAZY)
+    @ManyToOne
     private Movie movie;
-    @ManyToOne(fetch = FetchType.LAZY)
+    @ManyToOne
```

Comment:
Should we actually use lazy initialization and use join fetch or something on the MovieSessionDao?